src/audio/pipeline/pipeline-stream.c

```diff
@@ -459,51 +459,25 @@ int pipeline_trigger_run(struct pipeline *p, struct comp_dev *host, int cmd)
 	return ret;
 }
 
-/* Walk the graph to active components in any pipeline to find
```

Comment:
to clarify more we have a ```pipeline->timing_dai``` pointer.

Reply:
Is it guaranteed that we have a single DAI per pipeline? Could be there dailess pipelines?

Reply:
> Is it guaranteed that we have a single DAI per pipeline? 

I think for DMA driven pipeline timing this would be a yes for timing reasons (otherwise we would have a timing conflict). We would need a primary timing DAI, although others DAIs could be connected in the pipeline via ASRCs

> Could be there dailess pipelines?

Yes, I suppose so, but they would not be DMA driven.

I've no issue with the PR, was just thinking about an optimization but I think it seems this PR is for both timer and DMA driven pipelines.